tests/functional/test_download.py

```diff
@@ -139,7 +139,7 @@ def test_download_vcs_link(script):
     It should allow -d flag for vcs links, regression test for issue #798.
     """
     result = script.pip(
-        'install', '-d', '.', 'git+git://github.com/pypa/pip-test-package.git'
+        'download', '-d', '.', 'git+git://github.com/pypa/pip-test-package.git'
```

Comment:
I'm wondering whether we should duplicate all those tests to keep checking for the 2 next versions that the existing`pip install --download ...` commands keep working along with the new `pip download` command...